docs/dashboards-new/filter-template-variables.md

```diff
@@ -51,12 +51,16 @@ To add a Custom List variable to a dashboard, do the following:
 
 1. In the **Create Template Variable** panel, enter a unique **Variable Name**. Spaces and special characters, with the exception of an underscore (_), are not allowed in value names.
 
-![Create Template Variable dialog.png](/img/dashboards-new/filter-template-variables/Create-Template-Variable-dialog.png)
+    ![Create Template Variable dialog.png](/img/dashboards-new/filter-template-variables/Create-Template-Variable-dialog.png)
 
 1. Select **Custom List** as the **Variable Type**.
 1. **List Items** are your variable values. Use a comma separated list for variable options, separating individual options with a comma. For example, `small, medium, large`.
-1. Optional: Provide a **Default Value** for the variable.
-1. Optional: Toggle ON **Include the option to select all values (\*) **to include a wildcard asterisk (\*) in the available options.
+1. (Optional) **Default Value**. Provide a **Default Value** for the variable.
+1. (Optional) **Include the option to select all values**. Toggle ON to include a wildcard asterisk (\*) in the available options.
```

Comment:
Can you remove this line. We'd decided to remove this toggle altogether. You may also have to update some of your screenshots.